utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/universal/UniversalSchemaRegistryClientFactory.scala

```diff
@@ -10,11 +11,12 @@ class UniversalSchemaRegistryClientFactory extends SchemaRegistryClientFactory {
 
   override type SchemaRegistryClientT = SchemaRegistryClient
 
-  override def create(config: SchemaRegistryClientKafkaConfig): SchemaRegistryClientT =
-    createConfluentFactory.create(config)
-
-  private def createConfluentFactory: SchemaRegistryClientFactory = {
-    CachedConfluentSchemaRegistryClientFactory
+  override def create(config: SchemaRegistryClientKafkaConfig): SchemaRegistryClientT = {
+    if (config.kafkaProperties.get("schema.registry.url").exists(_.endsWith(".servicebus.windows.net"))) {
```

Comment:
now, one can't have SR behind proxy - maybe it's worth to write this in the docs?

Reply:
Good point, I wrote about that. If smb will have such case, we will introduce option to force specific schema registry implementation